.github/workflows/create-release-proposal.yml

```
@@ -0,0 +1,75 @@
+# This action requires the following secrets to be set on the repository:
+#   GH_USER_NAME: GitHub user whose Jenkins and GitHub token are defined below
+#   GH_USER_TOKEN: GitHub user token, to be used by ncu and to push changes
+#   JENKINS_TOKEN: Jenkins token, to be used to check CI status
+
+name: Create Release Proposal
+
+on:
+  workflow_dispatch:
+    inputs:
+      release-line:
+        required: true
+        type: number
+        default: 23
+        description: 'The release line (without dots or prefix). e.g: 22/20/18'
```

```suggestion
        description: 'The release line (without dots or prefix). e.g: 22'
```

this format is really confusing as it's making it look like a date 😅 it tripped me up while reading it